### DISCUSSION TITLE
HTML Report Total Test scenario count does not match with execution count

### DISCUSSION THREAD

**Discussion by 2vireddy** [getgauge/gauge#1850] [General]
After report execution the test scenario count in html report and execution details don't match. Please let me know if there are any settings to be changed.

![image](https://user-images.githubusercontent.com/29040810/104469759-dfc0fe80-55de-11eb-9476-6e79a00b314f.png)

![image](https://user-images.githubusercontent.com/29040810/104469918-11d26080-55df-11eb-87db-6455e6f19178.png)

**Comment by sriv** [comment]
I am guessing that you have table driven execution? If you do, could you please check if it's driving a spec or scenario?

Also, what is the number that you see in other reports, ex. xml-report?

**Reply by 2vireddy** [reply]
@sriv Thanks for responding. 
Please find below the settings in default.env.  
# Allows data driven at scenario level
allow_scenario_datatable = true

Also there is a scenario which is data driven with 3 sets of data, i see that the count of passed scenarios does not consider the data driven condition where as the Total scenarios considers the data driven condition

**Reply by sriv** [reply]
Yup, so the total scenarios in html report is treating each row as a separate scenario, whereas everywhere else it is accounted as a single scenario.

**Comment by harimadusumilli** [comment]
I have a similar or may be the same issue.
I have a Data Driven table csv file that I am using and its treating all the three datasets as a single test passed, is this the way it is to be represented. Where as in the total scenarios it is treating it as 3. 
![Gauge_scenario_reporting](https://user-images.githubusercontent.com/25215979/105855723-62ba5f80-5fb6-11eb-87e1-79e5f0eb0945.PNG)

Can you make the total scenarios match the count of passed + failed + skipped.
